ui/app/styles/components/tabs.scss

```diff
@@ -11,7 +11,9 @@
   box-shadow: inset 0 -1px 0 $grey-light;
 
   ul {
```

Comment:
thanks for putting the properties in alphabetical order! makes it easier to find + read!